index.html

```diff
@@ -117,6 +124,299 @@ <h2>I'm a software developer</h2>
+            <div class="tag-buttons">html</div>
+          </div>
+
+          <a class="see-projects" href="#"><div class="enable-button">See Project</div></a>
+        </div>
+      </div>
+
+      <div class="image-placeholder">
+        <div class="text-placeholder">
+          <h4 class="project-title">
+            Multi-Post Stories Gain+Glory
+          </h4>
+
+          <div class="tag">
+            <div class="tag-buttons">Ruby on rails</div>
+            <div class="tag-buttons">css</div>
+            <div class="tag-buttons">JavaScript</div>
+            <div class="tag-buttons">html</div>
+          </div>
+
+          <a class="see-projects" href="#"><div class="enable-button">See Project</div></a>
+        </div>
+      </div>
+
+      <div class="image-placeholder">
+        <div class="text-placeholder">
+          <h4 class="project-title">
+            Multi-Post Stories Gain+Glory
+          </h4>
+
+          <div class="tag">
+            <div class="tag-buttons">Ruby on rails</div>
+            <div class="tag-buttons">css</div>
+            <div class="tag-buttons">JavaScript</div>
+            <div class="tag-buttons">html</div>
+          </div>
+
+          <a class="see-projects" href="#"><div class="enable-button">See Project</div></a>
+        </div>
+      </div>
+
+      <div class="image-placeholder">
+        <div class="text-placeholder">
+          <h4 class="project-title">
+            Multi-Post Stories Gain+Glory
+          </h4>
+
+          <div class="tag">
+            <div class="tag-buttons">Ruby on rails</div>
+            <div class="tag-buttons">css</div>
+            <div class="tag-buttons">JavaScript</div>
+            <div class="tag-buttons">html</div>
+          </div>
+
+          <a class="see-projects" href="#"><div class="enable-button">See Project</div></a>
+        </div>
+      </div>
+
+      <div class="image-placeholder">
+        <div class="text-placeholder">
+          <h4 class="project-title">
+            Multi-Post Stories Gain+Glory
+          </h4>
+
+          <div class="tag">
+            <div class="tag-buttons">Ruby on rails</div>
+            <div class="tag-buttons">css</div>
+            <div class="tag-buttons">JavaScript</div>
+            <div class="tag-buttons">html</div>
+          </div>
+
+          <a class="see-projects" href="#"><div class="enable-button">See Project</div></a>
+        </div>
+      </div>
+
+      <section class="about-me">
+        <h2 class="primary-header about-me-header">About me</h2>
+        <p class="promo">
+          I can help you build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </p>
+
+        <a class="see-projects resume-button" href="#"
+          ><div class="enable-button resume-button">Get My Resume</div></a
+        >
```

Comment:
Please check my comment above on buttons to make changes to this

Reply:
I can't see any comment on the button